web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/Persondata.kt

```diff
@@ -83,16 +83,22 @@ object Persondata {
     data class Adresse constructor(
         val linje1: String,
         val linje2: String? = null,
-        val linje3: String? = null
+        val linje3: String? = null,
+        val registrert: LocalDateTime? = null,
+        val registrertAv: String? = null
```

Comment:
Kunne vi heller ha laget ett bitte-lite objekt for å holde styr på informasjon relatert til siste endring.
```kotlin
data class SistEndret(
        val ident: String,
        val tidspunkt: LocalDateTime,
        val system: String
)
```

Dette objektet kan da brukes på tvers av informasjonselementer (f.eks Telefon) slik at vi får en mer helhetlig opplevelse av dataene i frontendkoden.